programming-core/src/main/java/org/objectweb/proactive/core/httpserver/HTTPServer.java

```diff
@@ -122,12 +124,13 @@ public void stop() throws Exception {
         this.server.stop();
```

Comment:
It looks more coherent to me to shutdown the threadpool only when the server is destroyed.